bin/generate-lists.rb

```diff
@@ -68,7 +76,8 @@ def tags_for(db, zooniverse_id)
 
   species_to_track.each do |species|
     count = species_count(document, species)
-    if count != nil and classification_count > 0 and count >= (classification_count / 2)
+    blanks = blank_count(document)
+    if count != nil and classification_count > 0 and count >= ((classification_count - blanks) / 2)
```

Comment:
You might want to add `and blanks != nil` after `count != nil` too just in case one is blank and the other isn't -- at the moment that can't happen (`count` will always be `nil` when `blanks` is), but it'd be easy to miss this if the code changes in the future.